README.md

```diff
@@ -169,6 +169,27 @@ Locally:
 
 ## How-To
 
+### Add content to example page
+
+[The example page](https://www.ory.sh/docs/examples) provides an overview of Ory examples.
+Add a new example or modify existing entries:
+
+- Open `examples-content.tsx` under the path `docs/src/pages/_assets/`
+- Copy the following snippet and append it in the correct array (either official, community or self-hosting examples):
+
+```tsx
+{
+      title: 'Protect a Page with Login: NextJs/React', //Your example title goes here
+      language: 'typescript',                           //The main programming language of your example
```

Comment:
I think there's a limited number of options to choose from as they are tied to icons, right? If so, how about you write what are the options one can choose from without uploading a new icon or editing other files? :)

Reply:
oh yea good point. I have plans to update the icons a little bit and will add some docs for it at that point, does that seem ok? so sometime next week probably.

Reply:
That's okay although it wouldn't hurt to list the available options out of the gate. If that's a high-effort thing that you'll have to re-do in a couple of days then I'm okay with adding a list in a future PR.